megamek/i18n/megamek/client/messages.properties

```diff
@@ -2090,23 +2150,37 @@ PlanetaryConditionsDialog.labAtmosphere=Atmospheric Pressure
 PlanetaryConditionsDialog.BlowingSands=Blowing Sand
 PlanetaryConditionsDialog.EMI=EMI
 PlanetaryConditionsDialog.labFog=Fog
-PlanetaryConditionsDialog.labGrav=Gravity
+PlanetaryConditionsDialog.labGrav=Gravity (g)
 PlanetaryConditionsDialog.labLight=Light
-PlanetaryConditionsDialog.labTemp=Temperature
+PlanetaryConditionsDialog.labTemp=Temperature (deg C)
 PlanetaryConditionsDialog.labWeather=Weather
 PlanetaryConditionsDialog.labWind=Wind
-PlanetaryConditionsDialog.labMinWind=Mimimum Wind
-PlanetaryConditionsDialog.labMaxWind=Maximum Wind
+PlanetaryConditionsDialog.labMinWind=between
+PlanetaryConditionsDialog.labMaxWind=and
 PlanetaryConditionsDialog.NumberFormatError=Number Format Error
 PlanetaryConditionsDialog.title=Planetary Conditions
-PlanetaryConditionsDialog.EnterValidTemperature=Temperature must be between -200 and 200 degrees Celsius.
+
+PlanetaryConditionsDialog.invalid.integer=Enter an integer number.<BR>
+PlanetaryConditionsDialog.invalid.number=Enter a number.<BR>
+PlanetaryConditionsDialog.invalid.tempRange=The Temperature must be between -200 and 200 �C.<BR>
```

Comment:
```suggestion
PlanetaryConditionsDialog.invalid.tempRange=The Temperature must be between -200 and 200 \u00B0C.<BR>
```

Reply:
As above

Reply:
And yes the character Ive used here doesnt work either :)